docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -991,6 +991,9 @@ They can be used for prewhere optimization only if we enable `set allow_statisti
 
 #### Available Types of Column Statistics {#available-types-of-column-statistics}
 
+- `min_max`
```

Comment:
\+ newline after l. 996.

I complained in the past already about the name "Uniq" and lobbied to call it "HyperLogLog" instead. Anyways, it's all water under the bridge now (I am good).

We nevertheless ended up with a weird mix of camelCase and under_scores for the statistics names. Can we therefore please aim for a little consistency and rename "min_max" to "MinMax"?